README.md

```diff
@@ -39,27 +39,38 @@ See all targets with `make help`
 ### QuickStart Demonstration
 
 1. Get an API token exoscale.com
-1. `export EXOSCALE_API_KEY=<the-key>`
-1. `export EXOSCALE_API_SECRET=<the-secret>`
-1. `make local-install`
+2. `export EXOSCALE_API_KEY=<the-key>`
```

Comment:
No need to manually increment here. Most markdown renderers will do that for you.

Reply:
Well as you can see the previous markdown didn't work quite well lol :D

Reply:
The current readme shows fine for me though?

https://github.com/vshn/provider-exoscale/blob/master/README.md#quickstart-demonstration

Be aware that the source code will not automatically increment the numbering, it's only in the rendered version.

But anyway it's just a nitpick.